### PR TITLE
Noga get right sizing filter

### PR DESCRIPTION
**Pull request by nogakinor1**
fix parameters

### CODE REVIEW

File: api/services/ocean/ecs/paths/right-sizing-recommend.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 get:
```
Comment (sagishoshan) [inline api/services/ocean/ecs/paths/right-sizing-recommend.yaml:1]: @nogakinor1 I think this needs to be a POST call, according to the new instructions.
please check

**Review by sagishoshan — CHANGES_REQUESTED** [review]

File: api/services/ocean/ecs/paths/right-sizing-recommend.yaml (hunk 1 above)
Comment (nogakinor1) [reply api/services/ocean/ecs/paths/right-sizing-recommend.yaml:1]: @sagishoshan ecs is not relevant to the change, so I didn't change it. what do you think?

Comment (sagishoshan) [reply api/services/ocean/ecs/paths/right-sizing-recommend.yaml:1]: I think we need to check the actual routes to see what is working, and then have those calls aligned

Comment (nogakinor1) [reply api/services/ocean/ecs/paths/right-sizing-recommend.yaml:1]: @sagishoshan I checked with containers team, it should remain get

**Review by sagishoshan — APPROVED** [review]

**Pull request merged**